basename/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: basename
 title: Basename
 description: This plugin is used to get the last item of a file path or URL using Python's basename utility
-version: 1.0.2
+version: 2.0.0
```

Comment:
We can use a minor version bump on this one because `path` turns into `Path` in the product. First letter is capitalized if `title` is not present so the behavior is the same.